presto-raptorx/src/main/java/com/facebook/presto/raptorx/metadata/ShardCommitCleanerDao.java

```diff
@@ -80,4 +81,57 @@ void cleanupCreatedChunks(
     @SqlUpdate("DELETE FROM created_chunks WHERE chunk_id IN (<chunkIds>)")
     void deleteCreatedChunks(
             @BindList Set<Long> chunkIds);
+
+    // worker transactions
+
+    default void abortTransactions(Collection<Long> excludedTransactionIds, long endTime, long nodeId)
```

Comment:
Here all `transactions` are actually referring to the "worker" transaction (see my comments about renaming). We should avoid using the word `transaction` here because it refers to coordinator level transactions.

Reply:
I named it WorkerTransactions. 

Our transactions are just used to track created_chunks, same in V1. Coordinator track its created_chunks, and workers track its own created_chunks. I am not changing the word "Transaction" for now